docs/usage/configuration-options.md

```diff
@@ -2181,7 +2181,7 @@ Table with options:
 Post-upgrade tasks are commands that are executed by Renovate after a dependency has been updated but before the commit is created.
 The intention is to run any additional command line tools that would modify existing files or generate new files when a dependency changes.
 
-Each command must match at least one of the patterns defined in `allowedPostUpgradeCommands` (a global-only configuration option) in order to be executed.
+Each command must match at least one of the patterns defined in `allowedPeriUpgradeCommands` (a global-only configuration option) in order to be executed.
```

Comment:
| Term   | Meaning |
| ------ | --------|
| `peri` | Around  |
| `pre`  | Before  |

> `allowedPeriUpgradeCommands`

I think you meant allowed**Pre**UpgradeCommands? 😉 

There are more places where you use `peri`, so search and replace them if needed.

Reply:
No, because @rarkins said he prefers using a single config argument for commands allowed for pre-upgrade and post-upgrade tasks ([see conversation](https://github.com/renovatebot/renovate/pull/20423#discussion_r1108093615)).

So that's why I changed all `allowedPreUpgradeCommands` and `allowedPostUpgradeCommands` to `allowedPeriUpgradeCommands`. `peri` means `pre` and `post` in this context.

Reply:
If we want a single command for both `pre` and `post` phases then `peri` is good. 😉

Reply:
I think it's too obscure, we'd be better with `allowedUpgradeCommands`